maintenance/deleteBatch.php

```diff
@@ -39,6 +39,9 @@ public function __construct() {
 		$this->addOption( 'i', "Interval to sleep between deletions" );
 		$this->addArg( 'listfile', 'File with titles to delete, separated by newlines. ' .
 			'If not given, stdin will be used.', false );
+
+		// Wikia change
+		$this->addOption( 'by-id', 'Provided list contains article IDs instead of titles', false, false /* $withArgs */ );
```

Comment:
The `grep`'s `likewise` indicates that the translators were translating all help at once, which might not always be the case. That obviously does not apply here to any great extent. I am just mentioning my concern and what I find a good practice.